resource-management/test/resourceRegionMgrSimulator/app/simulator.go

```diff
@@ -30,12 +30,12 @@ import (
 )
 
 type RegionConfig struct {
-	RegionName            string
-	RpNum                 int
-	NodesPerRP            int
-	MasterPort            string
-	DataPattern           string
-	WaitTimeForMakeRpDown int
+	RegionName             string
+	RpNum                  int
+	NodesPerRP             int
+	MasterPort             string
+	DataPattern            string
+	WaitTimeForDataPattern int
```

Comment:
nit. naming, WaitTimeForDataChangePatterns?

Reply:
will change to new name.

Reply:
done.